module/VuFind/src/VuFind/Search/Factory/SolrReservesBackendFactory.php

```diff
@@ -55,24 +55,20 @@ public function __construct()
     }
 
     /**
-     * Create the SOLR backend.
+     * Create the record collection factory.
      *
-     * @param Connector $connector Connector
-     *
-     * @return \VuFindSearch\Backend\Solr\Backend
+     * @return RecordCollectionFactoryInterface
      */
-    protected function createBackend(Connector $connector)
+    protected function createRecordCollectionFactory()
```

Comment:
Is there a reason you didn't use `getCreateRecordCallback()` instead of the whole `createRecordCollectionFactory()` method here?

Reply:
No, I just forgot to check the factories after adding getCreateRecordCallback.